build/android/app/src/main/java/com/facebook/igl/sample/SampleActivity.java

```diff
@@ -33,6 +33,8 @@ public class SampleActivity extends Activity implements View.OnClickListener {
   private final int selectedTabColor = Color.BLUE;
   private final int unSelectedTabColor = Color.GRAY;
 
+  protected boolean mEnableStencilBuffer = false;
```

Comment:
Please separate the changes to existing files from the introduction of the new project.

Reply:
Done.
 I have moved the changes about SampleActivity.java into another PR(https://github.com/facebook/igl/pull/215).